internal/venice-consumer/src/main/java/com/linkedin/venice/client/consumer/VeniceChangelogConsumerClientFactory.java

```diff
@@ -69,25 +88,41 @@ public synchronized <K, V> VeniceChangelogConsumer<K, V> getChangelogConsumer(St
             newStoreChangelogClientConfig,
             consumer != null ? consumer : new KafkaConsumer<>(newStoreChangelogClientConfig.getConsumerProperties()));
       }
+      newStoreChangelogClientConfig.getInnerClientConfig().setMetricsRepository(metricsRepository);
```

Comment:
Just curious why we only set up for `metricsRepo` for After Image use case only? Besides, do we really need this class?

Reply:
So it's more in preparation.  We're going to need to add metrics pretty soon, and it seems simpler to get it wired in the short term to the client in offspring.

We right now have zero metrics, and I'm anticipating requests for metrics from clients.  Basically adding it here now saves me an extra PR later.

Reply:
done.